test/sequential/test-watch-mode.mjs

```diff
@@ -103,7 +104,7 @@ tmpdir.refresh();
 
 // Warning: this suite can run safely with concurrency: true
 // only if tests do not watch/depend on the same files
-describe('watch mode', { concurrency: true, timeout: 60_0000 }, () => {
+describe('watch mode', { concurrency: true, timeout: 60_000 }, () => {
```

Comment:
fwiw i like 60e3 for things like this, makes it so you don’t have to count the zeroes :-)